python-manual/modules/ROOT/pages/result-summary.adoc

```diff
@@ -160,29 +169,94 @@ print(summary.summary_notifications)
+
+        # The notification's classification is counterpart to `neo4j.NotificationCategory`.
+        # However, the term `category` has a different meaning in the context of GQL.
+        print("classification:", status.classification)
+        print("unparsed classification:", status.raw_classification)
+
+        print("severity:", status.severity)
+        print("unparsed severity:", status.raw_severity)
+
+    # Any raw extra information provided by the DBMS:
+    print("diagnostic record:", status.diagnostic_record)
+    print("=" * 80)
+"""
+GQLSTATUS: 02000
+description: note: no data
+is notification: False
+diagnostic record: {'OPERATION': '', 'OPERATION_CODE': '0', 'CURRENT_SCHEMA': '/'}
+================================================================================
+GQLSTATUS: 03N91
+description: info: unbounded variable length pattern. The provided pattern `(:Person {name: 'Alice'})-[*]->(:Person {name: 'Bob'})` is unbounded. Shortest path with an unbounded pattern may result in long execution times. Use an upper limit (e.g. `[*..5]`) on the number of node hops in your pattern.
+is notification: True
+position: line: 1, column: 22, offset: 21
+classification: NotificationClassification.PERFORMANCE
+unparsed classification: PERFORMANCE
+severity: NotificationSeverity.INFORMATION
+unparsed severity: INFORMATION
+diagnostic record: {'_classification': 'PERFORMANCE', '_status_parameters': {'pat': "(:Person {name: 'Alice'})-[*]->(:Person {name: 'Bob'})"}, '_severity': 'INFORMATION', '_position': {'offset': 21, 'line': 1, 'column': 22}, 'OPERATION': '', 'OPERATION_CODE': '0', 'CURRENT_SCHEMA': '/'}
+================================================================================
+"""
+----
+
+======
+=====
+
 
+[role=label--new-5.7]
 === Filter notifications
 
 By default, the server analyses each query for all categories and severity of notifications.
-Starting from version 5.7, you can use the parameters `notifications_min_severity` and/or `notifications_disabled_categories` to restrict the severity or category of notifications that you are interested into.
-You may disable notifications altogether by setting the minimum severity to `OFF`.
-You can use those parameters either when creating a `Driver` instance, or when creating a session.
-
+Starting from version 5.7, you can use the parameters `notifications_min_severity` and/or `notifications_disabled_categories`/`notifications_disabled_classifications` to restrict the severity and/or category/classification of notifications that you are interested into.
 There is a slight performance gain in restricting the amount of notifications the server is allowed to raise.
 
+The severity filter applies to both Neo4j and GQL notifications, whereas the category filter applies to Neo4j notifications only, and the classification filter applies to GQL notifications only.
+You can use any of those parameters either when creating a `Driver` instance, or when creating a session.
```

Comment:
This is incorrect. They are merged. Set either of them and notifications as well as GQL statuses will be affected. In fact, some drivers will error if you set both.

Reply:
Is the current recommendation fine (i.e. set one, whichever; example shows setting classification filter), and do you know if all drivers will apply either setting filter to either type form of notification?